src/Catalog/PackageCatalogItem.cs

```diff
@@ -21,13 +21,22 @@ public class PackageCatalogItem : AppendOnlyCatalogItem
         public DateTime? CreatedDate { get; }
         public DateTime? LastEditedDate { get; }
         public DateTime? PublishedDate { get; }
-
-        public PackageCatalogItem(NupkgMetadata nupkgMetadata, DateTime? createdDate = null, DateTime? lastEditedDate = null, DateTime? publishedDate = null, string licenseNames = null, string licenseReportUrl = null)
+        public PackageDeprecationItem Deprecation { get; }
+
+        public PackageCatalogItem(
+            NupkgMetadata nupkgMetadata, 
+            DateTime? createdDate = null, 
+            DateTime? lastEditedDate = null, 
+            DateTime? publishedDate = null, 
+            string licenseNames = null, 
+            string licenseReportUrl = null,
+            PackageDeprecationItem deprecation = null)
         {
             NupkgMetadata = nupkgMetadata;
             CreatedDate = createdDate;
             LastEditedDate = lastEditedDate;
             PublishedDate = publishedDate;
+            Deprecation = deprecation;
         }
 
         public override IGraph CreateContentGraph(CatalogContext context)
```

Comment:
Is there an UnitTest for this method?

Reply:
AFAIK the only testing around this is through the `FeedHelpers` tests that I edited below. The methods on `FeedHelpers` that are tested call this method. If I saw tests on this method alone I would have added them.

Reply:
I found some tests for the method. They seem more geared towards reading a `nupkg` from disk and asserting the output. Unfortunately, however, it seems substantially harder to add this additional logic to there. Given that I'm already testing the behavior in the other tests, I think it's fine not to add it here.